packages/nextjs-template/utils/build.ts

```diff
@@ -97,12 +102,24 @@ export function getNoteMeta(id: string): Promise<NoteProps> {
 }
 
 let _CONFIG_CACHE: DendronConfig | undefined;
-export function getConfig(): Promise<DendronConfig> {
+export async function getConfig(): Promise<DendronConfig> {
```

Comment:
instead of defining this in `getConfig`, we should add the check in https://github.com/dendronhq/dendron/blob/fix/publish-nullish-publishing-config-field-runtime-error/packages/pods-core/src/builtin/NextjsExportPod.ts#L492:L492

its during the building stage that the config is inspected and acted upon

Reply:
Yes this extra parsing should not be necessary since it is already done in `NExtjsExportPod`. I removed that change in nextjs-template [`27cb739` (#3649)](https://github.com/dendronhq/dendron/pull/3649/commits/27cb739d6762447403630e42b3016e4059d401bb)